### ISSUE TITLE
wrong 	totalBytes*Rate

### ISSUE THREAD

**Issue by illbrain** [pthoelken/fritzbox-zabbix-monitoring#16]
Hi there,
somehow the values for totalBytesReceivedRate and totalBytesSendRate are wrong. I download at 250Mbit/s and the output in zabbix is:
fritz.box totalBytesReceivedRate 42s 29.52 MBit/s

**Comment by pthoelken** [comment]
Thanks for your report. I'll take a look into it as soon as I can. Thanks.

**Comment by pthoelken** [comment]
Did you check the formula behind the item? 

Here is a breakdown of the formula:

- max(//totalBytesReceived,3m): This expression returns the maximum value of the number of bytes received over a 3 minute period.
- min(//totalBytesReceived,3m): This expression returns the minimum value of the number of bytes received over a period of 3 minutes.
- (max(//totalBytesReceived,3m)-min(//totalBytesReceived,3m)): This expression calculates the difference between the maximum and minimum value of the number of bytes received over a period of 3 minutes.
/ 120: This division by 120 is done to obtain the average bandwidth utilisation per second.

To summarise, this formula calculates the average bandwidth usage per second over a period of 3 minutes.

Maybe this helps.

**Comment by illbrain** [comment]
ok, If I divide by 12 it works. 
so i closed the case.
